dydx/dydxViews/dydxViews/Shared/dydxSliderInputView.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  dydxSliderInputView.swift
```

Comment:
@ruixhuang want your thoughts here as well in case you object. This file introduces a pattern where we split the view model into a view model and a (usually private) corresponding `View` struct. The `View` struct will enable us to take advantage of SwiftUI APIs which depend on the context being a struct. The class `ViewModel` will enable us to continue using the viewbuilder/viewModel pattern